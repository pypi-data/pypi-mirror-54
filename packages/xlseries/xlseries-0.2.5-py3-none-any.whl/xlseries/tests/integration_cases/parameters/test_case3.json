{
    "continuity": true, 
    "time_multicolumn": false, 
    "time_composed": true, 
    "series_names": null, 
    "missing_value": null, 
    "time_header_coord": "A4", 
    "data_ends": 119, 
    "frequency": "Q", 
    "data_starts": 7, 
    "blank_rows": false, 
    "headers_coord": [
        "B4", 
        "C4", 
        "D4"
    ], 
    "missings": false, 
    "alignment": "vertical", 
    "time_alignment": 0
}