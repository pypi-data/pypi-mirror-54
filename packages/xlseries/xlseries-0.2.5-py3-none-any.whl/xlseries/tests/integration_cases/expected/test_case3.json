{"Indice de precios de exportacion":{"504921600000":69.8,"512697600000":72.7,"520560000000":74.5,"528508800000":73.4,"536457600000":70.6,"544233600000":74.2,"552096000000":79.6,"560044800000":80.0,"567993600000":80.7,"575856000000":85.2,"583718400000":91.7,"591667200000":93.7,"599616000000":91.7,"607392000000":90.4,"615254400000":90.0,"623203200000":88.3,"631152000000":89.5,"638928000000":84.4,"646790400000":87.0,"654739200000":91.5,"662688000000":84.4,"670464000000":84.7,"678326400000":86.1,"686275200000":87.5,"694224000000":86.1,"702086400000":86.9,"709948800000":89.2,"717897600000":86.6,"725846400000":85.2,"733622400000":85.8,"741484800000":88.6,"749433600000":85.1,"757382400000":83.2,"765158400000":87.6,"773020800000":90.0,"780969600000":89.9,"788918400000":93.3,"796694400000":95.2,"804556800000":95.4,"812505600000":92.3,"820454400000":94.3,"828316800000":102.3,"836179200000":101.6,"844128000000":98.2,"852076800000":97.3,"859852800000":97.5,"867715200000":95.4,"875664000000":93.3,"883612800000":87.5,"891388800000":87.3,"899251200000":83.7,"907200000000":81.6,"915148800000":74.6,"922924800000":75.0,"930787200000":76.4,"938736000000":80.7,"946684800000":84.0,"954547200000":83.3,"962409600000":85.9,"970358400000":86.9,"978307200000":86.4,"986083200000":80.0,"993945600000":81.0,"1001894400000":76.4,"1009843200000":75.1,"1017619200000":75.5,"1025481600000":79.8,"1033430400000":81.0,"1041379200000":87.0,"1049155200000":84.0,"1057017600000":85.4,"1064966400000":90.4,"1072915200000":98.4,"1080777600000":102.5,"1088640000000":99.2,"1096588800000":99.3,"1104537600000":98.7,"1112313600000":101.8,"1120176000000":107.6,"1128124800000":109.9,"1136073600000":111.0,"1143849600000":112.9,"1151712000000":117.1,"1159660800000":119.7,"1167609600000":123.9,"1175385600000":123.3,"1183248000000":128.6,"1191196800000":145.7,"1199145600000":166.2,"1207008000000":166.4,"1214870400000":174.4,"1222819200000":152.4,"1230768000000":139.1,"1238544000000":145.4,"1246406400000":150.1,"1254355200000":154.0,"1262304000000":158.4,"1270080000000":151.3,"1277942400000":155.8,"1285891200000":171.0,"1293840000000":185.2,"1301616000000":190.5,"1309478400000":191.0,"1317427200000":188.3,"1325376000000":189.7,"1333238400000":193.5,"1341100800000":196.8,"1349049600000":195.1,"1356998400000":194.2,"1364774400000":194.2,"1372636800000":188.2,"1380585600000":183.2,"1388534400000":187.7},"Indice de precios de importacion":{"504921600000":94.9,"512697600000":97.6,"520560000000":97.0,"528508800000":95.8,"536457600000":95.8,"544233600000":91.9,"552096000000":89.2,"560044800000":100.5,"567993600000":100.5,"575856000000":98.5,"583718400000":95.7,"591667200000":109.2,"599616000000":109.5,"607392000000":107.6,"615254400000":109.1,"623203200000":115.5,"631152000000":110.3,"638928000000":111.8,"646790400000":112.3,"654739200000":116.9,"662688000000":114.5,"670464000000":112.0,"678326400000":105.2,"686275200000":105.3,"694224000000":104.4,"702086400000":102.7,"709948800000":104.1,"717897600000":105.1,"725846400000":104.6,"733622400000":104.0,"741484800000":100.5,"749433600000":103.8,"757382400000":104.7,"765158400000":102.3,"773020800000":103.2,"780969600000":106.9,"788918400000":106.6,"796694400000":106.1,"804556800000":106.4,"812505600000":108.8,"820454400000":112.3,"828316800000":106.8,"836179200000":104.4,"844128000000":108.5,"852076800000":109.4,"859852800000":106.3,"867715200000":104.8,"875664000000":105.6,"883612800000":103.9,"891388800000":100.5,"899251200000":98.3,"907200000000":97.9,"915148800000":97.7,"922924800000":95.9,"930787200000":96.3,"938736000000":97.3,"946684800000":100.2,"954547200000":98.6,"962409600000":96.7,"970358400000":98.2,"978307200000":98.5,"986083200000":96.8,"993945600000":93.7,"1001894400000":92.0,"1009843200000":94.4,"1017619200000":88.3,"1025481600000":89.3,"1033430400000":89.4,"1041379200000":92.2,"1049155200000":92.0,"1057017600000":91.9,"1064966400000":94.0,"1072915200000":95.6,"1080777600000":99.1,"1088640000000":101.7,"1096588800000":102.5,"1104537600000":105.5,"1112313600000":108.5,"1120176000000":107.9,"1128124800000":108.9,"1136073600000":111.2,"1143849600000":112.7,"1151712000000":115.3,"1159660800000":115.3,"1167609600000":116.2,"1175385600000":119.9,"1183248000000":125.6,"1191196800000":123.7,"1199145600000":126.8,"1207008000000":141.9,"1214870400000":146.5,"1222819200000":131.9,"1230768000000":120.7,"1238544000000":122.9,"1246406400000":121.7,"1254355200000":119.3,"1262304000000":122.7,"1270080000000":129.7,"1277942400000":127.6,"1285891200000":126.4,"1293840000000":131.9,"1301616000000":140.3,"1309478400000":139.0,"1317427200000":131.3,"1325376000000":130.4,"1333238400000":144.4,"1341100800000":132.4,"1349049600000":127.9,"1356998400000":136.6,"1364774400000":147.2,"1372636800000":139.5,"1380585600000":133.0,"1388534400000":138.0},"Indice de terminos del intercambio":{"504921600000":73.5511064278,"512697600000":74.487704918,"520560000000":76.8041237113,"528508800000":76.617954071,"536457600000":73.6951983299,"544233600000":80.7399347116,"552096000000":89.2376681614,"560044800000":79.6019900498,"567993600000":80.2985074627,"575856000000":86.4974619289,"583718400000":95.8202716823,"591667200000":85.8058608059,"599616000000":83.7442922374,"607392000000":84.0148698885,"615254400000":82.4931255729,"623203200000":76.4502164502,"631152000000":81.1423390752,"638928000000":75.4919499106,"646790400000":77.4710596616,"654739200000":78.2720273738,"662688000000":73.711790393,"670464000000":75.625,"678326400000":81.8441064639,"686275200000":83.0959164292,"694224000000":82.4712643678,"702086400000":84.6153846154,"709948800000":85.6868395773,"717897600000":82.3977164605,"725846400000":81.4531548757,"733622400000":82.5,"741484800000":88.1592039801,"749433600000":81.9845857418,"757382400000":79.4651384909,"765158400000":85.6304985337,"773020800000":87.2093023256,"780969600000":84.0972871843,"788918400000":87.5234521576,"796694400000":89.72667295,"804556800000":89.6616541353,"812505600000":84.8345588235,"820454400000":83.9715048976,"828316800000":95.7865168539,"836179200000":97.3180076628,"844128000000":90.5069124424,"852076800000":88.9396709324,"859852800000":91.7215428034,"867715200000":91.0305343511,"875664000000":88.3522727273,"883612800000":84.2155919153,"891388800000":86.8656716418,"899251200000":85.1475076297,"907200000000":83.3503575077,"915148800000":76.3561924258,"922924800000":78.2064650678,"930787200000":79.3354101765,"938736000000":82.9393627955,"946684800000":83.8323353293,"954547200000":84.4827586207,"962409600000":88.8314374354,"970358400000":88.4928716904,"978307200000":87.7157360406,"986083200000":82.6446280992,"993945600000":86.4461045891,"1001894400000":83.0434782609,"1009843200000":79.5550847458,"1017619200000":85.5039637599,"1025481600000":89.3617021277,"1033430400000":90.6040268456,"1041379200000":94.3600867679,"1049155200000":91.3043478261,"1057017600000":92.9270946681,"1064966400000":96.170212766,"1072915200000":102.9288702929,"1080777600000":103.4308779011,"1088640000000":97.5417895772,"1096588800000":96.8780487805,"1104537600000":93.5545023697,"1112313600000":93.8248847926,"1120176000000":99.7219647822,"1128124800000":100.9182736455,"1136073600000":99.8201438849,"1143849600000":100.1774622893,"1151712000000":101.5611448395,"1159660800000":103.81613183,"1167609600000":106.6265060241,"1175385600000":102.8356964137,"1183248000000":102.3885350318,"1191196800000":117.7849636217,"1199145600000":131.072555205,"1207008000000":117.2656800564,"1214870400000":119.0443686007,"1222819200000":115.5420773313,"1230768000000":115.2444076222,"1238544000000":118.3075671277,"1246406400000":123.336072309,"1254355200000":129.0863369656,"1262304000000":129.0953545232,"1270080000000":116.6538164996,"1277942400000":122.1003134796,"1285891200000":135.2848101266,"1293840000000":140.4094010614,"1301616000000":135.7804704205,"1309478400000":137.4100719424,"1317427200000":143.412033511,"1325376000000":145.4754601227,"1333238400000":134.0027700831,"1341100800000":148.6404833837,"1349049600000":152.5410476935,"1356998400000":142.1669106881,"1364774400000":131.9293478261,"1372636800000":134.9103942652,"1380585600000":137.7443609023,"1388534400000":136.0144927536}}